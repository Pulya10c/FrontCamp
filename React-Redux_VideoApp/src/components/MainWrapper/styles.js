import styled from 'styled-components';
import { mediaMd, mediaSm } from 'utils/style-utils';

export const MainWrapper = styled.main`
  margin: 0 auto;
  max-width: 1240px;
  padding: 5px;

  ${mediaMd`
     max-width: 720px;
  `};

  ${mediaSm`
     max-width: 550px;
  `};
`;
