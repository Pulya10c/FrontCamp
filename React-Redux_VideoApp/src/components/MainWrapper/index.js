import { MainWrapper } from './styles';

export default MainWrapper;
