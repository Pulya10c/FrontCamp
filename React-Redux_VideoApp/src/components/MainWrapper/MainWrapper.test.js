import React from 'react';
import { shallow } from 'enzyme';
import MainWrapper from './';

describe('Component MainWrapper', () => {
  it('should render correctly', () => {
    expect(shallow(<MainWrapper />)).toMatchSnapshot();
  });
});
