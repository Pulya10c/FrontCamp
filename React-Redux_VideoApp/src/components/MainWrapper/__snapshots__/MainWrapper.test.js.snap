// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component MainWrapper should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  margin: 0 auto;
  max-width: 1240px;
  padding: 5px;

  ",
          "
  @media (max-width: ",
          "768",
          "px) {
    ",
          "
     max-width: 720px;
  ",
          ";
  }
",
          ";

  ",
          "
  @media (max-width: ",
          "576",
          "px) {
    ",
          "
     max-width: 550px;
  ",
          ";
  }
",
          ";
",
        ],
      },
      "displayName": "styled.main",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "main",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
